### PR TITLE
Add OpenSpades

### PR DESCRIPTION
**Pull request by digiberk**
OpenSpades is an open-source clone of Ace of Spades

I am not the original developer of this project but because no one has added it to Flathub, I thought I would go ahead and do it. The flatpak manifest file(s) are in the openspades branch. If anything is missing, some other files are [here](https://github.com/yvt/openspades/tree/master/flatpak). 

Credit goes to @kdeleteme and @theunknownxy for creating the manifest file. (https://github.com/yvt/openspades/blob/master/flatpak/jp.yvt.OpenSpades.json)

### CODE REVIEW

**Comment by digiberk** [comment]
bot, build jp.yvt.OpenSpades

**Comment by kdeleteme** [comment]
Hi @digiberk. There's a mistake that I made in the manifest which may have caused the build failure.

```
          "url": "https://imagemagick.org/download/ImageMagick.tar.gz",
          "sha256": "c9e6723aad729140d4af0771d780d20823ca5b0c2fc52cf5a9b2028e923d33d4"
```

The *url* is targeting the latest imagemagick version while the *sha256sum* matches with a release several versions behind. Setting the *url* specific to a version (perhaps the latest) along with its corresponding *sha256sum* will correct that.

I'd also recommend adding the [shared-modules](https://github.com/flathub/shared-modules) as a git submodule, as **glew** depends on that.

**Comment by digiberk** [comment]
Thanks for the quick reply @kdeleteme ! This is all new to me so I'll see how it goes. I have updated the sha256 values now.

**Comment by digiberk** [comment]
bot, build jp.yvt.OpenSpades

**Comment by digiberk** [comment]
bot, build jp.yvt.OpenSpades

File: jp.yvt.OpenSpades.json [hunk 1]
```diff
@@ -52,7 +52,7 @@
         {
           "type": "archive",
           "url": "https://imagemagick.org/download/ImageMagick.tar.gz",
```
Comment (theunknownxy) [inline jp.yvt.OpenSpades.json]: You definitely want to change the URL to a specific version, too. If you don't do that, the build will fail in the future when imagemagick releases a new version.

There is directory of releases here: https://imagemagick.org/download/releases/

Comment (digiberk) [reply jp.yvt.OpenSpades.json]: Thanks for the heads up! I will definitely do that.

**Comment by digiberk** [comment]
bot, build jp.yvt.OpenSpades

**Comment by digiberk** [comment]
bot, build jp.yvt.OpenSpades

**Comment by digiberk** [comment]
Closed.

**Comment by digiberk** [comment]
I have created a new PR, apologies for the inconvenience. New one can be viewed here: https://github.com/flathub/flathub/pull/930

**Pull request closed without merging**